ff/src/fields/mod.rs

```diff
@@ -67,6 +67,31 @@ macro_rules! field_new {
     };
 }
 
+#[macro_export]
+macro_rules! field_new_from_raw_repr {
+    ($name:ident, $c0:expr) => {
+        $name {
+            0: $c0,
+            1: core::marker::PhantomData,
+        }
+    };
+    ($name:ident, $c0:expr, $c1:expr $(,)?) => {
+        $name {
+            c0: $c0,
+            c1: $c1,
+            _parameters: core::marker::PhantomData,
+        }
+    };
+    ($name:ident, $c0:expr, $c1:expr, $c2:expr $(,)?) => {
+        $name {
+            c0: $c0,
+            c1: $c1,
+            c2: $c2,
+            _parameters: core::marker::PhantomData,
+        }
+    };
+}
```

Comment:
Where is this used? Also, how is it different from the existing `field_new`

Reply:
Ah I see, you don't convert to montgomery form here. Is there a reason the original constants can't be in normal (not Montgomery) form?